mautrix_facebook/web/segment_analytics.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import os
+
+from yarl import URL
+import aiohttp
+
+from ..user import User
+
+log = logging.getLogger("mau.web.public.analytics")
+http: aiohttp.ClientSession
+host: str = "api.segment.io"
+
+try:
+    segment_key = os.environ["SEGMENT_API_KEY"]
+    http = aiohttp.ClientSession()
+    log.debug("Enabled segment analytics tracking")
+except (ImportError, KeyError):
+    segment_key = None
```

Comment:
I don't think that log will go anywhere since it happens before the logger is configured, but it's probably not necessary anyway. Also moved the ClientSession init to `else` so it only catches the actual environment keyerror

```suggestion
except KeyError:
    segment_key = None
else:
    http = aiohttp.ClientSession()
```